etc/client.api.report.api.md

```diff
@@ -9,7 +9,6 @@ import type { ActionParameterDefinition } from '@osdk/api';
 import type { BBox } from 'geojson';
 import type { BrandedApiName } from '@osdk/api';
 import type { InterfaceDefinition } from '@osdk/api';
-import type { IsNever as IsNever_2 } from 'type-fest';
```

Comment:
what does renaming the types do that fixes the API leaks?

Reply:
Nothing. Moving type-fest to a regular dependency is the real fix.